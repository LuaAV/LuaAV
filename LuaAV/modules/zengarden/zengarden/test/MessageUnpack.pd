#N canvas 332 142 450 300 10;
#X obj 21 19 loadbang;
#X msg 21 113 hello;
#X msg 254 113 one 11 42;
#X msg 174 113 11 one two;
#X msg 101 113 a 1 2 3 4;
#X msg 64 113 13;
#X obj 21 143 unpack s f 0 wookie f;
#X obj 258 170 print a;
#X obj 198 170 print b;
#X obj 138 170 print c;
#X obj 78 170 print d;
#X obj 21 170 print e;
#X obj 79 19 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 342 135 unpack;
#X obj 21 40 t b b b b b b;
#X msg 342 112 2 1;
#X obj 342 170 print z;
#X connect 0 0 14 0;
#X connect 1 0 6 0;
#X connect 2 0 6 0;
#X connect 3 0 6 0;
#X connect 4 0 6 0;
#X connect 5 0 6 0;
#X connect 6 0 11 0;
#X connect 6 1 10 0;
#X connect 6 2 9 0;
#X connect 6 3 8 0;
#X connect 6 4 7 0;
#X connect 12 0 14 0;
#X connect 13 0 16 0;
#X connect 13 1 16 0;
#X connect 14 0 1 0;
#X connect 14 1 5 0;
#X connect 14 2 4 0;
#X connect 14 3 3 0;
#X connect 14 4 2 0;
#X connect 14 5 15 0;
#X connect 15 0 13 0;
