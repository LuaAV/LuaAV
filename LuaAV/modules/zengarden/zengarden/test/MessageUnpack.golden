[@ 0.000ms] z: 1
[@ 0.000ms] z: 2
[@ 0.000ms] c: 42
[@ 0.000ms] d: 11
[@ 0.000ms] e: one
[@ 0.000ms] a: 4
[@ 0.000ms] b: 3
[@ 0.000ms] c: 2
[@ 0.000ms] d: 1
[@ 0.000ms] e: a
[@ 0.000ms] e: hello
